glide.yaml

```diff
@@ -13,6 +13,8 @@ import:
   version: dev
 - package: go.uber.org/thriftrw
   version: dev
+- package: github.com/anmitsu/go-shlex
+- package: github.com/jessevdk/go-flags
```

Comment:
are these really necessary? we don't import them? why did you need to add them? should they be in testImports instead?